core/amber/src/main/python/pytexera/udf/examples/rudf/r_tuple_operator.py

```diff
@@ -0,0 +1,56 @@
+# Note: make sure R path is initialized in udf.conf and make sure that
+# the following packages (in R) are installed: coro, arrow
+# --- Source Operator Examples ---
+r_tuple_source_zero_tuple = """
+library(coro)
+coro::generator(function() {
+    yield (list())
+    # yield (NULL) works too
+    })
+"""
+
+r_tuple_source_one_tuple = """
+library(coro)
+coro::generator(function() {
+    yield (list(
+            col1 = 1L, # R integer
+            col2 = "A", # R string
+            col3 = TRUE, # R logical (boolean)
```

Comment:
use `attribute` or `attr` for short instead of `col` for naming convention.

Reply:
Fixed

Reply:
Fixed